modules/news/news.graphql.js

```diff
@@ -13,7 +13,7 @@ const newsInput = `
 input NewsInput {
 title: [LanguageInput]
 text: [LanguageInput]
-images: [PrimaryImageInput]
+images: PrimaryImageInput
```

Comment:
Why have you made this change?

Reply:
It's ok. I asked him to do so because it will be only one image and we don't need an array here